src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = {};
+
+  for (const key in robot) {
+    if (reverseRobot.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      reverseRobot[`${robot[key]}`] = key;
+    }
```

Comment:
Мне кажется, что [`${robot[key]}`] необязательно записывать с помощью интерполяции.

Reply:
Agree, you don't need string here at all, it's redundant. Also you don't need else here
```suggestion
      return null;
    }
    
    reverseRobot[robot[key]] = key;
```